test/device.js

```diff
@@ -8,18 +8,31 @@ const uaIgnore = [
   'Mozilla/5.0 (PlayBook; U; RIM Tablet OS 2.1.0; en-US) AppleWebKit/536.2+ (KHTML, like Gecko) Version/7.2.1.0 Safari/536.2+',
   'Mozilla/5.0 (Linux; U; Android 2.2; en-gb; GT-P1000 Build/FROYO) AppleWebKit/533.1 (KHTML, like Gecko) Version/4.0 Mobile Safari/533.1',
   'Mozilla/5.0 (hp-tablet; Linux; hpwOS/3.0.5; U; en-US) AppleWebKit/534.6 (KHTML, like Gecko) wOSBrowser/234.83 Safari/534.6 TouchPad/1.0',
-  'gsa-crawler (Enterprise; T5-JJQHLGRCX9WKD; ssc-aps-notification-docubridge@hc-sc.gc.ca)',
+  /gsa-crawler/,
   'UCWEB/2.0 (Java; U; MIDP-2.0; Nokia203/20.37) U2/1.0.0 UCBrowser/8.7.0.218 U2/1.0.0 Mobile',
   'Dorado WAP-Browser/1.0.0/powerplay/2',
-  'SCH-U365/1.0 NetFront/3.0.22.2.23 (GUI) MMP/2.0'
+  'SCH-U365/1.0 NetFront/3.0.22.2.23 (GUI) MMP/2.0',
+  /vivo\s+\w+\s+Build/
```

Comment:
The site reports that these vivo devices should be tablets, but the user agent strings that this pattern matches are [actually phones](http://www.vivo.com/en/). Just googling the model pulls that up. I think they got miscategorized to target the [vivo tablet](https://www.asus.com/us/Tablets/Vivo-Tab-Products/)